kubernetes/cpi-inner.yml

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
Not sure I understand. How would you remove cpi url, make cpi version latest, remove static IPs, etc.?